include/cookie.php

```diff
@@ -34,13 +34,34 @@ protected static function parse_args( $args ) {
 		$defaults = array(
 			'expires'  => 0 !== $expiration ? time() + $expiration : 0,
 			'path'     => COOKIEPATH,
-			'domain'   => COOKIE_DOMAIN, // Cookie domain must be set to false for localhost ( default value for COOKIE_DOMAIN ) thanks to Stephen Harris.
+			'domain'   => COOKIE_DOMAIN, // Cookie domain must be set to false for localhost (default value for `COOKIE_DOMAIN`) thanks to Stephen Harris.
 			'secure'   => is_ssl(),
 			'httponly' => false,
 			'samesite' => 'Lax',
 		);
 
-		return wp_parse_args( $args, $defaults );
+		$args = wp_parse_args( $args, $defaults );
+
+		/**
+		 * Filters the Polylang cookie arguments.
+		 * /!\ This filter may be fired *before* the theme is loaded.
+		 *
+		 * @since 3.7
```

Comment:
I believe that we can push this now:
```suggestion
		 * @since 3.6
```